konfig

```diff
@@ -84,28 +85,46 @@ merge() {
 }
 
 import_ctx() {
+    declare -a tmpcfgs=()
     local tmpcfg
+    local tmpinputcfg=""
     local out=""
     local arg=""
-    if [[ "$1" == '-p' || "$1" == '--preserve-structure' ]]; then
-      arg="$1"
-      shift
-    fi
-    if [[ "$1" = '--save' || "$1" = '-s' ]]; then
-      out="${XDG_CACHE_HOME:-$HOME/.kube}/config"
-      shift
-    elif [[ "$1" =~ ^-(.*) ]]; then
-      error "unrecognized flag \"$1\""
-    fi
+    for OPT in "$@"; do
```

Comment:
I like this loop btw. It's idiomatic and much more readable!

Reply:
Thanks! I'm glad to hear that.